fabric-item-groups-v0/src/testmod/java/net/fabricmc/fabric/test/client/item/group/ItemGroupTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.client.item.group;
+
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.client.itemgroup.FabricItemGroupBuilder;
+
+@Environment(EnvType.CLIENT)
+public class ItemGroupTest implements ClientModInitializer {
+	public static final ItemGroup TEST_ITEM_GROUP = FabricItemGroupBuilder.create(new Identifier("fabric-item-groups-v0-testmod:test_itemgroup")).icon(() -> new ItemStack(Items.MINECART_WITH_COMMAND_BLOCK)).appendItems((list) -> Items.DYE.registerSubItems(Items.DYE, null, list)).build();
+
+	@Override
+	public void onInitializeClient() {
```

Comment:
might want to make a test item to add it to the tab?

Reply:
no

Reply:
should probs go in a content registries testmod